pipelines/api-pipelines.yml

```diff
@@ -108,26 +108,6 @@ stages:
               downloadType: 'single'
               artifactName: 'drop'
               downloadPath: '$(Build.ArtifactStagingDirectory)'
-
-          - task: AzureCLI@1
-            displayName: Get Cosmosdb URI and Key
-            inputs:
-              azureSubscription: $(azureSubscription)
-              scriptLocation: inlineScript
-              inlineScript: |
```

Comment:
Removes this inline script in favor for a less complex option to manage the url/keys within the variable group.  New forthcoming PR will allow for construction based on different environments.